micro-ui/web/micro-ui-internals/packages/modules/workbench/src/pages/employee/MDMSEdit.js

```
@@ -4,131 +1,192 @@
     config: {
-      enabled: moduleName && masterName && true,
-      select: (data) => { 
-        const uniqueFields = data?.SchemaDefinitions?.[0]?.definition?.["x-unique"]
-        const updatesToUiSchema = {}
-        uniqueFields.forEach(field => updatesToUiSchema[field] = {"ui:readonly":true})
-        return {schema:data?.SchemaDefinitions?.[0],updatesToUiSchema}
+      enabled: !!data && localizationCodes.length > 0,
+      select: (respData) => {
+        let messageMap = {};
+        if (Array.isArray(respData?.messages)) {
+          respData.messages.forEach((msg) => (messageMap[msg.code] = msg.message));
```

Biome (1.9.4)

[error] The assignment should not be in an expression.

The use of assignments in expressions is confusing.
Expressions are often considered as side-effect free.

(lint/suspicious/noAssignInExpressions)